REVIEW_CHECKLIST.rst

```diff
@@ -0,0 +1,96 @@
+================
+Review Checklist
+================
+
+Use this checklist to review pull requests. If the pull request is large,
+consider giving partial feedback quickly rather than taking a long time to
+review comprehensively.
+
+Large PRs
+---------
+- Consider giving partial feedback. The following are in order of magnitude
+  changes they will request, so rather than going down the list for one file,
+  prefer making a full pass for each section.
```

Comment:
Weird wording here. An alternative wording  suggestion:
```suggestion
- Consider giving partial feedback. The following items are ordered according to the magnitude
  of changes they will involve, so rather than going down the list for one file,
  prefer making a full pass for each section.
```

Reply:
A big improvement, thanks!